.github/pyinstaller/pyinstaller.spec

```diff
@@ -1,10 +1,40 @@
 # -*- mode: python -*-
 # Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
-import os.path
-import subprocess
```

Comment:
Removing `os` in favor of `pathlib`.
Removing `subprocess` because I believe it is unused.